src/types/positions.ts

```diff
@@ -100,6 +100,7 @@ export interface EarnDataProps {
   rewardsPositionIds?: string[]
   claimType?: ClaimType
   withdrawalIncludesClaim?: boolean
+  dailyYieldRatePercentage?: number
```

Comment:
can we document what goes in `yieldRates` and why we have a specific `dailyYieldRatePercentage`?